apps/web/components/AppListCard.tsx

```diff
@@ -85,6 +88,21 @@ export default function AppListCard(props: AppListCardProps) {
             </div>
           )}
         </div>
+        {credentialOwner && (
+          <div>
+            <Badge variant="gray">
+              <div className="flex items-center">
+                <Avatar
+                  className="mr-2"
+                  alt={credentialOwner.name || "Nameless"}
```

Comment:
Followup: We have nameless spread across the app. I think I confirmed elsewhere that name is a required field and it shouldn't be unset. But even if that's the case, we should introduce `function getName({name:string})`